project.clj

```diff
@@ -1,25 +1,25 @@
-(defproject kirasystems/saml20-clj "0.1.13-SNAPSHOT"
+(defproject juji/saml20-clj "0.1.13"
```

Comment:
This project name change is likely coming from the fork-only diff. I think we should keep the existing `kirasystems/saml20-clj` name.

Reply:
That's an oversight. Removed.